lib/utils/server/feature-flags.ts

```diff
@@ -1,17 +1,22 @@
-import { PostHog } from "posthog-node";
-
 export type FeatureFlag = "contributions_evolution_by_type" | "workspaces";
 
 export async function getAllFeatureFlags(userId: number) {
-  const client = new PostHog(
-    process.env.NEXT_PUBLIC_POSTHOG_ID as string,
+  const requestOptions = {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      api_key: process.env.NEXT_PUBLIC_POSTHOG_ID,
+      distinct_id: userId,
+    }),
+  };
 
-    { host: "https://app.posthog.com" }
+  const response = await fetch(
+    "https://app.posthog.com/decide?v=3", // or eu
```

Comment:
Non-blocking: what's `or eu`? 🤔

Reply:
Me not knowing how to type lol. I'll remove it.